war/pom.xml

```diff
@@ -450,7 +451,7 @@ THE SOFTWARE.
       <plugin>
         <groupId>org.eclipse.jetty</groupId>
         <artifactId>jetty-maven-plugin</artifactId>
-        <version>9.4.22.v20191022</version>
+        <version>9.4.37.v20210219</version>
```

Comment:
```suggestion
        <version>9.4.38.v20210224</version>
```

Reply:
Eh, this PR will conflict with yours anyway as we touch adjacent lines.